src/theme/OpenRPCDocIndex.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+const DocSidebar = require('@theme/DocSidebar').default;
+import { Sidebar } from '@docusaurus/plugin-content-docs/src/sidebars/types';
+import MarkdownDescription from '@metamask/open-rpc-docs-react/dist/MarkdownDescription/MarkdownDescription';
```

Comment:
ideally `@metamask/open-rpc-docs-react` exports this properly

Reply:
can we make an issue/Pr to track this? Maybe we should just do that before shipping this?

Reply:
made an issue here: https://github.com/MetaMask/open-rpc-docs-react/issues/28